docs/source/apps/aces/aces_qss.ipynb

```diff
@@ -0,0 +1,319 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports and API Token"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example tutorial notebook uses `qiskit-superstaq`, our Superstaq client for Qiskit,"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import qiskit\n",
+    "    import qiskit_superstaq as qss\n",
+    "except ImportError:\n",
+    "    print(\"Installing qiskit-superstaq...\")\n",
+    "    %pip install --quiet 'qiskit-superstaq[examples]'\n",
+    "    print(\"Installed qiskit-superstaq.\")\n",
+    "    print(\"You may need to restart the kernel to import newly installed packages.\")\n",
+    "    import qiskit\n",
+    "    import qiskit_superstaq as qss"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Required imports\n",
+    "import numpy as np\n",
+    "import itertools\n",
+    "\n",
+    "# Optional imports\n",
+    "import os  # Used if setting a token as an environment variable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To interface Superstaq via Qiskit, we must first instantiate a provider in `qiskit-superstaq` with `SuperstaqProvider()`. We then supply a Superstaq API token (or key) by either providing the API token as an argument of `qss.SuperstaqProvider()` or by setting it as an environment variable (see more details [here](https://superstaq.readthedocs.io/en/latest/get_started/basics/basics_qss.html#Set-up-access-to-Superstaq%E2%80%99s-API))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the qiskit superstaq provider for Superstaq backend\n",
+    "provider = qss.SuperstaqProvider()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ACES"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To submit an ACES job, you will need the following information:\n",
+    "* `target`: device that you want to characterize.\n",
+    "* `qubits`: indices of the qubits to characterize.\n",
+    "* `shots`: number of shots to use per circuit to run.\n",
+    "* `num_circuits`: number of random circuits to sample in order to get the gate eigenvalues.\n",
+    "* `mirror_depth`: for each circuit, how many mirrored moments to include.\n",
+    "* `extra_depth`: for each circuit, how many random moments to include.\n",
+    "* `method`: the type of execution method. If `method=\"noise-sim\"`, then optional arguments `noise` and `error_prob` must be passed as well.\n",
+    "\n",
```

Comment:
The bullet points don't seem to be building correctly

Reply:
In the notebook or when building the docs html?

Reply:
'''
If you just go to the notebook cell itself and replace it with this: To submit an ACES job, you will need the following information:

* `target`: device that you want to characterize.
* `qubits`: indices of the qubits to characterize.
* `shots`: number of shots to use per circuit to run.
* `num_circuits`: number of random circuits to sample in order to get the gate eigenvalues.
* `mirror_depth`: for each circuit, how many mirrored moments to include.
* `extra_depth`: for each circuit, how many random moments to include.
* `method`: the type of execution method. If `method="noise-sim"`, then optional arguments `noise` and `error_prob` must be passed as well.

With this information, you can submit an ACES job through the `submit_aces` method. This will take care of constructing the circuits needed to perform the protocol and will execute them. 
```

Docs will build with the proper formatting

Reply:
Done.